addons/web/models/models.py

```diff
@@ -966,11 +977,14 @@ def _search_panel_global_counters(self, values_range, parent_name):
         Note that we save the initial (local) counts into an auxiliary dict
         before they could be changed in the for loop below.
 
-        :param values_range: dict of the form
-            {
-                id: { 'id': id, '__count': c, parent_name: parent_id, ... }
-                ...
-            }
+        :param values_range: dict of the form:
```

Comment:
```suggestion
        :param dict values_range: has the form:
```

`Must` have the form?